medicines/doc-index-updater/src/create_manager/mod.rs

```diff
@@ -56,13 +56,15 @@ async fn try_process_from_queue(
                     == "Couldn't retrieve file: [-31] Failed opening remote file".to_string()
                 {
                     tracing::info!("Updating state to errored and removing message");
-                    let _ = state_manager.set_status(
-                        retrieval.message.job_id,
-                        JobStatus::Error {
-                            message: "Couldn't find file".to_string(),
-                            code: "404".to_string(),
-                        },
-                    );
+                    let _ = state_manager
+                        .set_status(
+                            retrieval.message.job_id,
+                            JobStatus::Error {
+                                message: "Couldn't find file".to_string(),
+                                code: "404".to_string(),
+                            },
+                        )
+                        .await?;
```

Comment:
I found that `.await?` was required to set the status in the delete process.

Reply:
Ah yeah - noticed this yesterday when I was looking into and completely forgot to do anything with it!